package.json

```diff
@@ -30,8 +30,8 @@
     "gh-pages": "2.0.0",
     "gulp-htmllint": "0.0.16",
     "jest": "^27.3.1",
-    "node-sass": "^4.14.1",
     "parcel": "^1.12.4",
+    "sass": "*",
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use git add src/.